### ISSUE TITLE
identify 'feat_dim'

### ISSUE THREAD

**Issue by Autoom** [TaeSoo-Kim/TCNActionRecognition#1]
Hi，I am now in Keras , and the variable' feat_dim'  which I cant understand , could you describe it ?  Thank you

**Comment by TaeSoo-Kim** [comment]
Autoom,

feat_dim refers to the dimension of the per-frame feature. For this particular dataset (NTURGBD-skeletons), feat_dim should be 150 per frame (at most 2 actors, 25 joints per actor, 3 coordinates per joint = 150).

Does this make sense?

**Comment by vamagoul** [comment]
Hello,

In process_skeleton.py the split in training and test samples is determined by 'S' letter in skeleton name, Instead, it should change to 'P' for cross_subject and 'C' for cross_view evaluation. The 'S' denotes setup and doesn't correspond to any evaluation setting. With the current setting the results will be inferior by 2%. However, if properly splitting the train and test set, the reported results from the paper are correct (tested with Keras 2.1)

**Comment by TaeSoo-Kim** [comment]
Yes, this is a known issue. I believe the convention must have changed since when I first wrote this code (as indicated by the consistent results.) Thanks for letting us know!

**Comment by JingxuanHou** [comment]
@vamagoul 
Hello, I am trying to reproduce the results from the paper as well, using the updated NTU RGB+D dataset and Keras 2.1. But I fail to report the same results and I cannot figure out why.  For the updated dataset, did you too modify the parameters like samplers_per_epoch, samples_per_validation and train_x_mean in the original code? Or did you change something else? Thank you! (Could you send me your modified code if it doesn't bother you too much? houjx14@mails.tsinghua.edu.cn)

**Comment by TaeSoo-Kim** [comment]
I have not tried the same experiments with the updated NTU+Keras 2.1. I am curious to see what do results look like with the new setup as well.
